documentation/examples/prometheus-kubernetes.yml

```diff
@@ -18,14 +18,14 @@ scrape_configs:
   bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
 
   kubernetes_sd_configs:
-  - masters:
+  - api_servers:
     - 'https://kubernetes.default.svc'
     in_cluster: true
 
   relabel_configs:
   - source_labels: [__meta_kubernetes_role]
     action: keep
-    regex: ^(?:master|node)$
+    regex: ^(?:apiserver|node)$
```

Comment:
So switch to just `apiserver|node` in this case? Feels weird tbh. Didn't expect the non-capturing wrap in there, just the anchors.